tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -314,6 +314,11 @@ class WskBasicUsageTests
             }, waitBeforeRetry = Some(1.second), N = 5)
     }
 
+    it should "include CLI user agent headers with outbound requests" in {
+        val stdout = wsk.cli(Seq("action", "list", "-v", "--auth", wskprops.authKey) ++ wskprops.overrides).stdout
```

Comment:
i recommend using the wsk.action.list function.  to do that, augment the ListOrGetFromCollection list() function with a `verbose: Boolean = false` parameter and explicitly pass that to the underlying cli() call.